cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1374,6 +1380,12 @@ public void run() {
         if (_executionTimerInvolveBrokerRemovalOrDemotion != null) {
           _executionTimerInvolveBrokerRemovalOrDemotion.update(duration, TimeUnit.MILLISECONDS);
         }
+
+        OPERATION_LOG.info("Execution Finished: task Id: {}; removed brokers: {}; demoted brokers: {}; total time used: {}ms.",
```

Comment:
nit: s/Finished/finished/  and please insert "successfully" as well

Where do interrupted or failed executions get logged? We should also track the time spent on those since, for example, a human-interrupted self-healing could mean we miss our Nimbus SLA.

Reply:
Fixed the nit but didn't add successfully. (It could also be failure or interruptions)

I believe this place will always be reached regardless of failure or interruption.

Reply:
@groelofs do you have any other concerns about this one?

Reply:
Do you have a way to distinguish success/failure? That's absolutely critical info, so I'd like it included. (The other logging location does indicate this.)

Also, since you're adding it here rather than the other location, can you include the UUID?

Reply:
> That's absolutely critical info, so I'd like it included. 

Good point. I have included that piece of information in the latest commit. 

The latest commit will show exceptions if execution failed, but it doesn't show interruptions. I think if we want to also include interruptions in the log, I'll prefer to just using the existing log lines + execution duration. The drawback is it will miss the information of whether it's demote/remove brokers.